src/components/BasicNodeViewer/BasicNodeViewer.scss

```diff
@@ -0,0 +1,43 @@
+@import '../../styles/mixins.scss';
+
+.basic-node-viewer {
```

Comment:
extracted from full-node-viewer without any changes in styles, only in classes structure